sample/src/app/messaging/messaging.component.ts

```diff
@@ -25,8 +24,15 @@ export class MessagingComponent implements OnInit {
   message$: Observable<any>;
   showRequest = false;
 
-  constructor(public readonly messaging: AngularFireMessaging, swPush: SwPush) {
-    messaging.usePublicVapidKey('BIDPctnXHQDIjcOXxDS6qQcz-QTws7bL8v7UPgFnS1Ky5BZL3jS3-XXfxwRHmAUMOk7pXme7ttOBvVoIfX57PEo').then(() => {
+  constructor(public readonly messaging: AngularFireMessaging, readonly swpush: SwPush) {
+    swpush.messages.subscribe(it => console.log('swpush', it));
```

Comment:
Was this just for debugging?

Reply:
Oh this is in the sample app, so this looks good then! Missed that on the first pass.

Reply:
Yeah, it seems FCM has made some changes to the FCM API that have affected us. Experimenting with different variations on Service Workers and FCM in the sample app & will probably revamp those guides shortly.